Frontend/src/app/views/dashboard/project-details/project-details.component.ts

```diff
@@ -87,9 +87,6 @@ export class ProjectDetailsComponent {
 
         this.teamDataSource = this.dataSourceBuilder.create(this.teamMembers);
         this.loadingProject = false;
-      },
-      (error) => {
-        this.router.navigateByUrl("/dashboard/projects");
```

Comment:
This is a much bigger problem and this work around only fixes the 1 example instance.